src/controllers/words.js

```diff
@@ -67,10 +67,7 @@ export const getWords = async (req, res, next) => {
     let words;
     let query;
     if (searchWord.match(/".*"/) || searchWord.match(/'.*'/)) {
-      const searchWordWithoutQuotes = searchWord
-        .split('')
-        .filter((c) => c !== '"' && c !== "'")
-        .join('');
+      const searchWordWithoutQuotes = searchWord.replaceAll(/[',"]/, '');
```

Comment:
I would double check this regex, but it's also going to target and remove commas